088-rebalance-progress-status.md

```diff
@@ -0,0 +1,374 @@
+# Partition Rebalance Progress Status
+
+This proposal introduces a new feature to monitor the progression of an ongoing partition rebalance executed by a Strimzi-managed Cruise Control instance via a `KafkaRebalance` custom resource. 
+
+## Current Situation
+
+At this time, Strimzi users are able to execute partition rebalances via `KafkaRebalance` custom resources but can only monitor the progression of those partition rebalances in two ways:
+
+- Manually querying the Cruise Control REST API endpoint directly. 
```

Comment:
Don't we lock these down with a Network Policy? Would the user have to alter the default set up to get access?

Reply:
Yes there are a couple of things that would need special configuration to enable a user to access the CC REST API directly

Reply:
I thought that the user API is now supported after the proposal to manage the users?

Please keep in mind that the network policies are there to allow the operator access and not just to prevent users from accessing it. (In environments where the network plugin denies-by-default, we need to create the network policy to make sure the operator can talk with the CC API)

Reply:
> I thought that the user API is now supported after the proposal to manage the users?

It is. What what I mean by "special configuration" here is that in order to hit the Cruise Control API endpoint directly, a person will need to alter the default Cruise Control setup - Add Cruise Control API user to authenticate, create `NetworkPolicy` to access Cruise Control API port from outside pod in denies-by-default environments, etc

Reply:
It requires creating a custom REST API user or disabling HTTP Basic authentication, I'll add a note here.

A custom `NetworkPolicy` is only required in denies-by-default environments, mentioning the custom API user requirement should drive home the point without being too pedantic.